mkosi/__init__.py

```diff
@@ -592,7 +607,7 @@ def complete_step(text: str, text2: Optional[str] = None) -> Generator[List[Any]
                 Callable[[CommandLineArguments, str, bool], None],
                 Callable[[CommandLineArguments, str, bool, bool], None],
             ]
-)
+            )
```

Comment:
Not that this is at the moment an issue, but this is the difference between black and flake8 and at some point I would like to see black as consistent formatting in this codebase and I'd rather switch this warning off than accomodate flake8 here.